src/main/java/nl/tudelft/ti2206/group9/gui/ExternalTicker.java

```diff
@@ -48,7 +51,14 @@ private void renderScene() {
 			final Group entities = renderEntities();
 			GameScreen.addWorld(entities);
 		}
+		
+		GameScreen.addOverlay(renderScore());
+	}
 
+	/**
+	 * @return VBox with score labels
+	 */
+	private VBox renderScore() {
```

Comment:
Nice addidition, but it isn't really part of this issue.

< review, organisation